index.html

```diff
@@ -54,7 +54,192 @@ <h1 class="title">
+            </p>
+            <p class="connect-link">
+                LET'S CONNECT
+            </p>
+            <ul class="social-media">
+                <li>
+                    <a href="https://github.com/Rimuru-Tempest-26" target="_blank"><img src="images/icons/github.svg"
+                            alt="Github icon"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/phone-khaing-hein-759497216/" target="_blank"><img
+                            src="images/icons/linkedin.svg" alt="Linkedin icon"></a>
+                </li>
+                <li>
+                    <img src="images/icons/peace.svg" alt="Peace icon">
+                </li>
+                <li>
+                    <a href="https://twitter.com/PhoneKhaingHein" target="_blank"><img src="images/icons/twitter.svg"
+                            alt="Twitter icon"></a>
+                </li>
+                <li>
+                    <a href="mailto:pkh2662003@gmail.com" target="_blank"><img src="images/icons/gmail.svg"
+                            alt="Gmail icon"></a>
+                </li>
+            </ul>
+            <button class="aboutme-button">Get my resume</button>
+        </div>
+        <div id="skills">
+            <ul>
+                <li class="skill-title">
+                    <div class="skill-list">
+                        Languages
+                        <img src="images/icons/down-arrow.svg" alt="Down arrow">
+                    </div>
+                    <div class="skill-detail">
+                        <img src="images/icons/javascript.svg" alt="Javascript image">
+                        <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a>
+                    </div>
+                    <div class="skill-detail">
+                        <img src="images/icons/html.svg" alt="HTML image">
+                        <a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a>
+                    </div>
+                    <div class="skill-detail">
+                        <img src="images/icons/css.svg" alt="CSS image">
+                        <a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a>
+                    </div>
+                </li>
+                <li class="skill-title" style="margin-top: 24px;">
+                    <div class="skill-list">
+                        Frameworks
+                        <img src="images/icons/right-arrow.svg" alt="Down arrow">
+                    </div>
+                </li>
+                <hr>
+                <li class="skill-title">
+                    <div class="skill-list">
+                        Skills
+                        <img src="images/icons/right-arrow.svg" alt="Down arrow">
+                    </div>
+                </li>
+                <hr style="margin-bottom: 90px;">
+            </ul>
+        </div>
+    </section>
+    <section id="contact-us">
```

Comment:
You did awesome on the whole project 👏🏽 but I had to mention this one although the latest section is for another project, you should consider adding some margin between this section `About` and the `Contact` section, this would improve your design a lot. This is what you have for now. 👇🏽 
![Screenshot 2023-02-18 at 09 16 51](https://user-images.githubusercontent.com/45739963/219870681-6e49f39b-f005-47e7-9654-8824d277ffb0.png)

Reply:
Fixed.